qcengine/tests/test_procedures.py

```diff
@@ -94,6 +94,23 @@ def test_berny_stdout(input_data):
     assert "All criteria matched" in ret.stdout
 
 
+@using("psi4")
+@using("berny")
+def test_berny_failed_gradient_computation(input_data):
+
+    input_data["initial_molecule"] = qcng.get_molecule("water")
+    input_data["input_specification"]["model"] = {"method": "HF", "basis": "sto-3g"}
+    input_data["input_specification"]["keywords"] = {"badpsi4key": "badpsi4value"}
+    input_data["keywords"]["program"] = "psi4"
+
+    input_data = OptimizationInput(**input_data)
+
+    ret = qcng.compute_procedure(input_data, "berny", raise_error=False)
```

Comment:
```suggestion
    input_model = OptimizationInput(**input_data)

    ret = qcng.compute_procedure(input_model, "berny", raise_error=False)
```

"model" for pydantic obj and "data" for dict is my convention. but my aim is merely a dummy commit to trigger CI.